common/authorization/token_key_provider.go

```diff
@@ -39,4 +41,11 @@ type TokenKeyProvider interface {
 	Close()
 }
 
+// RawTokenKeyProvider is a TokenKeyProvider that provides keys for validating JWT tokens
```

Comment:
Need to add comments that this is an optional alternative version of token key provider. However, if provider implements this interface, it will take precedent and shadow the implementation of TokenKeyProvider and be used to validate JWT token.

Reply:
If `RawTokenKeyProvider is a TokenKeyProvider`, shall we embed `TokenKeyProvider` inside `RawTokenKeyProvider` interface and only specify additional methods?

Reply:
I think it's fine as it is and there's no need to add `TokenKeyProvider` methods to `RawTokenKeyProvider`.